tools/test.py

```diff
@@ -150,11 +150,14 @@ def main():
         if args.out:
             print(f'\nwriting results to {args.out}')
             mmcv.dump(outputs, args.out)
-        kwargs = {} if args.options is None else args.options
+        kwargs = cfg.get('evaluation', {})
+        kwargs.pop('interval', None)
```

Comment:
@druzhkov-paul I did changes that you proposed. I remove ```interval``` key from ```kwargs``` as far as it used to define the frequency of evaluating and datasets did not know how to process it. Does it look nice?

Reply:
Ok, cool. Could you please remove `gpu_collect` key (if any) as well. It's used for evaluation hook in a distributed setup:

https://github.com/openvinotoolkit/mmdetection/ote/mmdet/core/evaluation/eval_hooks.py#L53

Reply:
Add rebase the branch, please.